### ISSUE TITLE
Disable GitHub Pages

### ISSUE THREAD

**Issue by ylemkimon** [Khan/KaTeX#1755]
https://github.com/Khan/khan.github.io/pull/10 has been merged and the old GitHub Pages and now lives at khan.github.io.

**Comment by kevinbarabash** [comment]
According to https://help.github.com/articles/unpublishing-a-project-pages-site/ it sounds like we have to delete the branch.  Seems like a weird way to stop publishing gh-pages.

**Comment by ylemkimon** [comment]
@kevinbarabash I've archived the branch at https://github.com/KaTeX/old-gh-pages.

**Comment by kevinbarabash** [comment]
@ylemkimon good idea.  Thanks for doing that.  I've deleted the `gh-pages` branch in this repo and Github Pages for this repo is no longer active.

<img width="770" alt="screen shot 2018-10-31 at 11 20 53 pm" src="https://user-images.githubusercontent.com/1044413/47830746-9e2c3c00-dd63-11e8-8fe1-b52291670b0e.png">
